contracts/libraries/CircleCCTPAdapter.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "../external/interfaces/CCTPInterfaces.sol";
+
+abstract contract CircleCCTPAdapter {
```

Comment:
Nit: prefer naming this to CircleCCTP_SpokePool since its extended by SpokePools and not used as an L1 "Adapter" just because this would be confuinsg with the contracts in `chain-adapters`

Reply:
Hmm agreed but could we use `CircleCCTPConnector` or something similar? The `CircleCCTPAdapter` is currently used by the chain adapters & the spoke pools since the logic is the same